README.md

```diff
@@ -1 +1,10 @@
-# clandry_project
\ No newline at end of file
+# clandry_project
+
+<img src="https://avatars.githubusercontent.com/u/86985765?v=4?s=100" width="100px;" alt=""/>
+  <href="https://github.com/catherinelandry">
+
+  <br /><sub><b>Catherine Landry (she/her)</b></sub>
+   
+   <br /><sub><b> I am currently doing my master's degree in psychology at the Université de Montréal. My wide range of interests is reflected through my eclectic research background. I have worked with spinal cord injured rats, ASD children and ended up working with functional connectivity following deafness. As of today, I'm particularly hyped about neuroimaging and coding, hence my learning journey with PSY6983. </b></sub>
```

Comment:
So I suggest to move this text above, outside of the HTML. 

Currently the HTML formatting is a bit off because the text is much longer than what the HTML was designed for.